### PR TITLE
feat: add additional release rules

### PR DESCRIPTION
**Pull request by mad-it**
fixes: #52

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -30,7 +30,14 @@ const parseOptions = {
...
  */
 const releaseRulesExtension = [
+  { release: 'patch', type: 'ci' },
   { release: 'patch', scope: 'deps', type: 'chore' },
+  {
+    message: '*typescript*',
+    release: 'patch',
+    scope: 'deps-dev',
+    type: 'chore',
+  },
```
Comment (acazacu) [inline src/index.ts]: Just checking, this ensures that all `chore(deps-dev): *typescript*` messages would trigger a release.

File: src/index.ts [hunk 2]
```diff
@@ -30,7 +30,14 @@ const parseOptions = {
  * https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-angular#type
  */
 const releaseRulesExtension = [
+  { release: 'patch', type: 'ci' },
```
Comment (acazacu) [inline src/index.ts]: What drove ci types (re)triggering a release?

File: src/index.ts (hunk 1 above)
Comment (acazacu) [reply src/index.ts]: Never mind, saw the Pull Request description.

File: src/index.ts (hunk 2 above)
Comment (acazacu) [reply src/index.ts]: Never mind, saw the Pull Request description.

File: src/index.ts (hunk 1 above)
Comment (mad-it) [reply src/index.ts]: correct, `*` is a wild card.

Comment (merlinnot) [reply src/index.ts]: So chores don't trigger releases? Why? There might be tons of dev dependencies that affect outputs: faro backend TypeScript and ncc, but look at the front end: literally hundreds of dependencies that change the deliverable.

Comment (mad-it) [reply src/index.ts]: Currently only `dev-deps` dont trigger releases. That's why this rule is added explicitly to allow typescript to release.

Comment (merlinnot) [reply src/index.ts]: But why those updates don't trigger releases? They should. It is needed.

**Comment by mad-it** [comment]
In order to fix the issue where the release rules are tailored  too much to Dott's needs, I will close this PR in favor of making a new PR in the future where the release rules are loosened up for the community while making it configurable for our own needs. 

The following link is the internal tracking issue. https://emtransit.atlassian.net/browse/CP-1610

**Pull request closed without merging**